apps/e2e-test/README.md

```diff
@@ -0,0 +1,114 @@
+# E2E-Test Framework
+
+The test automation framework for end-to-end testing with Cucumber-JS and Playwright. It abstracts browser interaction to simple steps that are written from a user perspective.
+
+## Prerequisites
+
+In order to run the automated test, you will need the following tools installed on your machine.
+
+- Node v16.16.0
```

Comment:
Should be `Nodejs`.

Reply:
Addressed the review comment: Updated to NodeJS